### DISCUSSION TITLE
Is LOCAL_APP still possible like it was on LTSP5

### DISCUSSION THREAD

**Discussion by Alex1130** [ltsp/ltsp#704] [Q&A] [answered]
I use this ltsp.conf to start a local app and use a local disk on the client, to minimize network traffic to the server:

```
[kamu]
LOCAL_APP = TRUE
LOCALDEV = TRUE
CONFIGURE_FSTAB= TRUE
RCFILE_01="/etc/00_start" #script that starts a local app on the client
CRONTAB_01="45 5    * * *   root /home/kamu/bin/remove_old_dirs/cleanup"
################
#Acer
################
[30:65:EC:85:03:3G]
LIKE=kamu
FSTAB_0="/dev/disk/by-uuid/a6008ac9-cad1-49b0-9dda-ddf446840b43 /home ext4 defaults  0 0"

```
I wonder if this is still an option? Or is there a new way of achieving this?

**Comment by alkisg** [comment, marked as answer]
All applications in the new LTSP are run locally, there's no need to specify "LOCALxx" anywhere.

To start a command as root, you can use `POST_SERVICE_xxx="your command"` in ltsp.conf.
To start a command as user after login, you can use the /etc/xdg/autostart directory.
